clients/fides-js/src/integrations/blueconic.ts

```diff
@@ -0,0 +1,55 @@
+declare global {
+  interface Window {
+    blueConicClient?: {
+      profile: {
+        setConsentedObjectives: (objectives: string[]) => void;
+        setRefusedObjectives: (objectives: string[]) => void;
+        updateProfile: () => void;
+      };
+      event: {
+        subscribe: any;
+      };
+    };
+  }
+}
+
+// https://support.blueconic.com/hc/en-us/articles/202605221-JavaScript-front-end-API
+const blueConicLoaded = () =>
+  typeof window.blueConicClient !== "undefined" &&
+  typeof window.blueConicClient.event !== "undefined" &&
+  typeof window.blueConicClient.event.subscribe !== "undefined";
+
+const configureObjectives = () => {
+  if (!blueConicLoaded() || !window.blueConicClient?.profile) {
+    return;
+  }
+
+  const profile = window.blueConicClient?.profile;
+  const { consent } = window.Fides;
+  if (consent.targeted_advertising_gpp_us_state) {
```

Comment:
I feel like I need to do something more expansive here. Should I be looking to do something like [this.](https://github.com/ethyca/fides/blob/42be590e3b64e566642d22b1c831bffa8cac4609/clients/fides-js/src/integrations/shopify.ts#L20)